Assets/Mirror/Runtime/ClientScene.cs

```diff
@@ -724,39 +716,31 @@ internal static void OnClientAuthority(NetworkConnection _, ClientAuthorityMessa
         }
 
         // called for the one object in the spawn message which is the local player!
-        internal static void OnSpawnMessageForLocalPlayer(uint netId)
+        internal static void OnSpawnMessageForLocalPlayer(NetworkIdentity identity)
         {
-            if (LogFilter.Debug) Debug.Log("ClientScene.OnSpawnMessageForLocalPlayer - " + readyConnection + " netId: " + netId);
+            if (LogFilter.Debug) Debug.Log("ClientScene.OnSpawnMessageForLocalPlayer - " + readyConnection + " netId: " + identity);
 
             // is there already an owner that is a different object??
             if (readyConnection.identity != null)
             {
                 readyConnection.identity.SetNotLocalPlayer();
             }
 
-            if (NetworkIdentity.spawned.TryGetValue(netId, out NetworkIdentity localObject) && localObject != null)
-            {
-                // this object already exists
-                localObject.connectionToServer = readyConnection;
-                localObject.SetLocalPlayer();
-                InternalAddPlayer(localObject);
-            }
+            localPlayer = identity;
```

Comment:
This code is unnecessary because CheckForLocalPlayer will be called after spawning